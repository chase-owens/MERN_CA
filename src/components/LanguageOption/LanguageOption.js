import React, { Fragment } from 'react';
import { theme } from '../../styles/theme';

const LanguagePicker = ({ name }) => {
  return (
    <Fragment style={{ width: '100%', height: '100%' }}>
      {
        <div>
          <h2>{name}</h2>
        </div>
      }
    </Fragment>
  );
};

export default LanguagePicker;
