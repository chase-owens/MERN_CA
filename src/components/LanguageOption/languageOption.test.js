import React from 'react';
import LanguageOption from './LanguageOption';
import { shallow } from 'enzyme';

describe('LanguageOption', () => {
  it('renders without crashing', () => {
    const languageOption = shallow(<LanguageOption />);
    expect(languageOption).toMatchSnapshot();
  });
});
