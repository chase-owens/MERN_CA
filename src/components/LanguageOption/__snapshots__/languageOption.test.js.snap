// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LanguageOption renders without crashing 1`] = `
<Fragment
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <div>
    <h2 />
  </div>
</Fragment>
`;
